runtime/shared_common/ByteDataManagerImpl.cpp

```diff
@@ -354,7 +354,6 @@ SH_ByteDataManagerImpl::find(J9VMThread* currentThread, const char* key, UDATA k
 	}	
 
 	Trc_SHR_BDMI_find_Exit(currentThread, resultCntr);
-		
```

Comment:
I see there are only blank line and copy right date change for this file. Please remove this file from this PR.